ibis/expr/types/temporal.py

```diff
@@ -994,3 +994,23 @@ def full_name(self):
             The name of the day of the week
         """
         return ops.DayOfWeekName(self._expr).to_expr()
+
+
+@public
+class TimeTravelTable:
```

Comment:
Note: I think there is a way to eliminate the `__init__` and `__getattr__` from `TimeTravelTable`. Will look in to this tomorrow.